frontend/sass/_wells.scss

```diff
@@ -2,16 +2,46 @@
 
 .well {
   background-color: $well-bg-color;
-  margin-top: 1.5em;
-  padding: 1em;
+  padding: 1.5rem;
```

Comment:
I think this `.well` class is used elsewhare, like on the site settings page. do these changes work there also?

Reply:
I'd like to remove the well from that page though. (In addition to a few other updates as well to the settings page). Should I go ahead an add that to this PR or leave it?

Reply:
I would do the settings updates in a separate PR. Is it ok to hold off on this change until that PR is in as well?

(and @jseppi I'm ok checking over this code on staging if we can merge early tonight or tomorrow so that we can check over staging before retro)

Reply:
I added the margin-bottom back in for now, so the spacing below each `well` on the settings page is preserved.  Should be good to go now!